### ISSUE TITLE
Style is not set on the Tab component

### ISSUE THREAD

**Issue by mikesmith1611** [facultyai/dash-bootstrap-components#151]
The style property that can be set on the Tab item is not used at all.

**Comment by tcbegley** [comment]
Hi @mikesmith1611 

It appears to be working for me, see this minimal example

```python
import dash
import dash_bootstrap_components as dbc

app = dash.Dash(external_stylesheets=[dbc.themes.BOOTSTRAP])

app.layout = dbc.Container(
    dbc.Tabs(
        [
            dbc.Tab("This is tab 1", label="tab 1"),
            dbc.Tab("This is tab 2", label="tab 2", style={"padding": "10px"}),
        ]
    )
)

if __name__ == "__main__":
    app.run_server(debug=True)
```

The padding is correctly applied to the second tab. This is happening [here](https://github.com/facultyai/dash-bootstrap-components/blob/master/src/components/tabs/Tabs.js#L80), the `Tabs` component is responsible for rendering tabs, the `Tab` component itself is really just used like an object and when used with `Tabs` will never have it's render method called.

If you have an example where it's not working, could you share it please so we can investigate?

**Comment by mikesmith1611** [comment]
Hi, my mistake. I wrongly thought that the style on the Tab component was supposed to be applied to the Li or nav-item that contains the label. I am using the Tabs in a callback so I do not set the children here so didn’t notice what was happening. It would be great if we had the ability to set a style for the nav-items. I would like them to all float left as I want to put them in a scroll bar in a situation where I have many tab items.

**Comment by tcbegley** [comment]
Ah understood! Yes, having some control over the style of the tab itself sounds like a good idea, and should be pretty doable. I'll make sure this gets added, thanks for the feedback!

**Comment by tcbegley** [comment]
Resolved by #152
